parcels/field.py

```diff
@@ -481,6 +481,9 @@ def reshape(self, data, transpose=False):
         if self.grid.lat_flipped:
             data = lib.flip(data, axis=-2)
 
+        if len(data.shape) == 1:
+            if self.grid.zdim > 1 and self.grid.xdim == 1 and self.grid.ydim == 1:
+                data = data[:, None, None]  # if vertical section
```

Comment:
why is the `data = data[:, None, None]` instead of `data = data[:, 1, 1]` ? Doesn't the non-notation cause issues when start computing the field on execute (throwing an index error in Field::computeTimeChunk) ?

Reply:
It's instead of `np.newaxis`, see e.g. https://stackoverflow.com/questions/26166471/3d-tiling-of-a-numpy-array

Reply:
Okay, I see. well, if that's pythonically fine, then I'm okay. It looks a but visually disturbing to me, but that's probably just different perspectives of C++ and Python coders. Test comply, so the fix is good to go.